app/uk/gov/hmrc/tariffclassificationfrontend/service/FileStoreService.scala

```diff
@@ -32,15 +32,34 @@ class FileStoreService @Inject()(connector: FileStoreConnector) {
 
   def getAttachments(c: Case)(implicit hc: HeaderCarrier): Future[Seq[StoredAttachment]] = {
     val attachmentsById: Map[String, Attachment] = c.attachments.map(a => a.id -> a).toMap
-    connector.get(c.attachments) map { files =>
-      files map { file =>
+    connector.get(c.attachments) map {
```

Comment:
can this code used below?

looks like the two methods have code in common.... we could extract it and define the common code only once

Reply:
Done